crates/verifier/src/groth16/converter.rs

```diff
@@ -15,6 +16,10 @@ use super::error::Groth16Error;
 /// The byte slice is represented as 2 uncompressed g1 points, and one uncompressed g2 point,
 /// as outputted from Gnark.
 pub(crate) fn load_groth16_proof_from_bytes(buffer: &[u8]) -> Result<Groth16Proof, Groth16Error> {
+    if buffer.len() < 256 {
```

Comment:
We should make these constants with informative names, rather than using numbers.